gallery/src/components/NxTextInput/NxTextInputDisabledExample.tsx

```diff
@@ -10,6 +10,6 @@ import { NxTextInput } from '@sonatype/react-shared-components';
 
 export default function NxTextInputDisabledExample() {
   return (
-    <NxTextInput placeholder="Disabled input" value="" isPristine={true} disabled/>
+    <NxTextInput placeholder="Disabled input" value="" isPristine={false} disabled/>
```

Comment:
Why this change?

Reply:
> Why this change?

It seemed odd to me that a disabled button started in a pristine state (though I agree it's no less odd than the other way), so I was curious if it made a difference to the styling, then I forgot that I did it.

Reply:
> > Why this change?
> 
> It seemed odd to me that a disabled button started in a pristine state (though I agree it's no less odd than the other way), so I was curious if it made a difference to the styling, then I forgot that I did it.

Reverted here: https://github.com/sonatype/sonatype-react-shared-components/pull/57/commits/a592c64696532f2e463a287af8e6805e72ef465c